prboom2/ICONS/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+if(UNIX AND NOT APPLE)
+    install(FILES dsda-doom.desktop DESTINATION share/applications)
+    install(FILES dsda-doom.svg DESTINATION share/icons/hicolor/scalable/apps)
```

Comment:
@fabiangreffrath Is an SVG icon enough? Or do we need to add a 128x128 PNG file?

Reply:
Svg is fine.